schema/patches/1.sql

```diff
@@ -1,21 +1,29 @@
-ALTER TABLE myury.act_permission SET SCHEMA myradio;
-ALTER TABLE myury.actions SET SCHEMA myradio;
-ALTER TABLE myury.api_class_map SET SCHEMA myradio;
-ALTER TABLE myury.api_key SET SCHEMA myradio;
-ALTER TABLE myury.api_key_auth SET SCHEMA myradio;
-ALTER TABLE myury.api_method_auth SET SCHEMA myradio;
-ALTER TABLE myury.award_categories SET SCHEMA myradio;
-ALTER TABLE myury.award_member SET SCHEMA myradio;
-ALTER TABLE myury.error_rate SET SCHEMA myradio;
-ALTER TABLE myury.menu_columns SET SCHEMA myradio;
-ALTER TABLE myury.menu_links SET SCHEMA myradio;
-ALTER TABLE myury.menu_module SET SCHEMA myradio;
-ALTER TABLE myury.menu_sections SET SCHEMA myradio;
-ALTER TABLE myury.menu_twigitems SET SCHEMA myradio;
-ALTER TABLE myury.modules SET SCHEMA myradio;
-ALTER TABLE myury.password_reset_token SET SCHEMA myradio;
-ALTER TABLE myury.photos SET SCHEMA myradio;
-ALTER TABLE myury.services SET SCHEMA myradio;
-DROP TABLE myury.services_versions;
-DROP TABLE myury.services_versions_member;
-DROP SCHEMA myury;
+BEGIN;
+
+CREATE TABLE jukebox.playlist_categories (
+    id SERIAL PRIMARY KEY,
+    name TEXT,
+    description TEXT
+);
+
+INSERT INTO jukebox.playlist_categories (id, name, description)
+VALUES (
+    1,
+    'General',
+    '<p>This category is for all playlists that don''t fit the others. They will not be played by Jukebox or Campus Playout.</p>'
```

Comment:
Either i'm reading this wrong, or is that quote not properly escaped? :)

Reply:
Nope, that's how postgres does quote escapes. Yeah it's ugly.